langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -208,6 +255,11 @@ public isolated function toJsonString(anydata v) returns string = @java:Method {
 #
 # Returns an error if the string cannot be parsed.
 #
+# ```ballerina
+# string student = "{\"index\": 12, \"name\": \"John\"}";
```

Comment:
Wouldn't it be more simple if a [backtick template](https://ballerina.io/learn/by-example/backtick-templates/) is used here?

```
string student = string `{"index": 12, "name": "John"}`;
```

Reply:
I also think it is a good way to do this.

Reply:
While it makes the example a little less complicated, the common use case IMO, is using `fromJsonString` with an arbitrary string. E.g., read/received. So I think the current form is a more real-world scenario.